Source/Core/TerrainEncoding.js

```diff
@@ -143,9 +144,21 @@ define([
          * @type {Boolean}
          */
         this.hasWebMercatorT = defaultValue(hasWebMercatorT, false);
+
+        /**
+         * Center of the 2.5D projected space. If defined, it is assumed that the projection requires 2D vertex attributes.
+         * @type {Cartesian3}
+         */
+        this.center2D = Cartesian3.clone(center2D);
+
+        /**
+         * The terrain mesh should contain projected positions for 2D space.
+         * @type {Boolean}
+         */
+        this.has2dPositions = defined(center2D);
```

Comment:
This naming convention is not consistent with `center2D`, which is not consistent with Cesium's use of camelCase.  Maybe this is a special case.  I dunno.  @lilleyse

Reply:
Same comment throughout.

Reply:
I'll switch to `hasPositions2D`